packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts

```diff
@@ -136,20 +118,11 @@ export class ZoweVsCodeExtension {
         return undefined;
     }
 
-    public static async inputBox(inputBoxOptions: vscode.InputBoxOptions): Promise<string> {
```

Comment:
same as the previous comment.

Just trying to prevent any possible disruptions before v3 (2024)